pycargoebuild/ebuild.py

```diff
@@ -79,6 +90,32 @@ def get_CRATES(crates: typing.Iterable[Crate],
                              if isinstance(c, FileCrate))) +
             "\n")
 
+def get_IUSE(features: typing.Optional[dict]) -> str:
+    """
+    Return the IUSE string for the given features dictionary.
+    """
+    if features is None:
+        return ""
+
+    default_features = features.get("default", [])
+
+    non_default_features = sorted(f"{feature} "
+                                  for feature in features.keys()
+                                  if feature != "default" and feature not in default_features)
+
+    default_features_str = " ".join(f"+{feature}" for feature in default_features)
+
+    return (default_features_str + " " + ''.join(non_default_features)).strip()
```

Comment:
```suggestion
    all_features = sorted(x for x in features if x != "default")
    return " ".join(f"+{x}" if x in default_features else x for x in all_features)
```

We want flags sorted by name, not `+` sign.